idea/testData/joinLines/stringTemplate/simple2.kt.after

```diff
@@ -0,0 +1,4 @@
+fun test2() {
+    val foo = "1"
+    val s = <caret>"${foo}bar"
```

Comment:
I think this is surprising behavior. Lines should be joined into a single expression only if both the left hand side and the right hand side of the concatenation are constants.

Reply:
Fixed